porcupine/plugins/run/terminal.py

```diff
@@ -116,16 +125,22 @@ def _run_in_x11_like_terminal(command: str, cwd: Path, env: dict[str, str]) -> N
         )
         return
 
-    real_command = [str(run_script), str(cwd), command]
-    real_command.extend(map(str, command))
-    subprocess.Popen([terminal, "-e", " ".join(map(shlex.quote, real_command))], env=env)
+    if command:
+        real_command = [str(run_script), str(cwd), command]
+        real_command.extend(map(str, command))
```

Comment:
`real_command.extend(map(str, command))` is dumb and weird:

```
>>> command = "python3 foo.py"
>>> real_command = ["/path/to/script.sh", "/home/akuli", command]
>>> real_command.extend(map(str, command))
>>> real_command
['/path/to/script.sh', '/home/akuli', 'python3 foo.py', 'p', 'y', 't', 'h', 'o', 'n', '3', ' ', 'f', 'o', 'o', '.', 'p', 'y']
>>> " ".join(map(shlex.quote, real_command))
"/path/to/script.sh /home/akuli 'python3 foo.py' p y t h o n 3 ' ' f o o . p y"
```

We can keep it in this PR because it was there already and running commands seems to work regardless.

Reply:
It went askew in https://github.com/Akuli/porcupine/commit/e44d4aae36030ffa65dbab3e9cff65e5fd6712a0